incubator/binding-risingwave.spec/src/main/streams/risingwave/create.table.with.primary.key/server.rpt

```diff
@@ -0,0 +1,95 @@
+# this file except in compliance with the License.  You may obtain a copy of the
+# License at
+#
+#   https://www.aklivity.io/aklivity-community-license/
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations under the License.
+#
+
+property serverAddress "zilla://streams/app0"
+
+accept ${serverAddress}
+    option zilla:window 8192
+    option zilla:transmission "duplex"
+
+accepted
+
+read zilla:begin.ext ${pgsql:beginEx()
+                              .typeId(zilla:id("pgsql"))
+                              .parameter("user", "root")
+                              .parameter("database", "dev")
+                              .parameter("application_name", "psql")
+                              .parameter("client_encoding", "UTF8")
+                              .build()}
+
+connected
+
+read zilla:data.ext ${pgsql:dataEx()
+                              .typeId(zilla:id("pgsql"))
+                              .query()
+                                .build()
+                              .build()}
+read "CREATE TOPIC IF NOT EXISTS balances (name VARCHAR, type VARCHAR, description VARCHAR, PRIMARY KEY (name));"
+     [0x00]
+
+write advise zilla:flush ${pgsql:flushEx()
+                              .typeId(zilla:id("pgsql"))
+                              .completion()
+                                .tag("CREATE_TABLE")
+                                .build()
+                              .build()}
+
+write advise zilla:flush ${pgsql:flushEx()
+                              .typeId(zilla:id("pgsql"))
+                              .ready()
+                                .status("IDLE")
+                                .build()
+                              .build()}
+
+write close
+read closed
+
+accepted
+
+read zilla:begin.ext ${pgsql:beginEx()
+                              .typeId(zilla:id("pgsql"))
+                              .parameter("user", "root")
+                              .parameter("database", "dev")
+                              .parameter("application_name", "psql")
+                              .parameter("client_encoding", "UTF8")
+                              .build()}
+
+connected
+
+read zilla:data.ext ${pgsql:dataEx()
+                              .typeId(zilla:id("pgsql"))
+                              .query()
+                                .build()
+                              .build()}
+read "CREATE TABLE IF NOT EXISTS balances(*, PRIMARY KEY (balance_id)) INCLUDE KEY AS balance_id WITH (connector='kafka', topic='balances', properties.bootstrap.server='localhost:9092', scan.startup.mode='latest', scan.startup.timestamp.millis='140000000' ) FORMAT UPSERT ENCODE AVRO ( schema.registry = 'http://localhost:8081');"
+     [0x00]
```

Comment:
```suggestion
read "CREATE TABLE IF NOT EXISTS balances(*, PRIMARY KEY (balance_id)) "
     "INCLUDE KEY AS balance_id "
     "WITH ("
         "connector='kafka', "
         "topic='balances', "
         "properties.bootstrap.server='localhost:9092', "
         "scan.startup.mode='latest', "
         "scan.startup.timestamp.millis='140000000' "
     ") "
     "FORMAT UPSERT ENCODE AVRO ( "
         "schema.registry = 'http://localhost:8081'"
     ");"
     [0x00]
```

 IT scripts are a bit like documentation, so they should also be formatted clearly.

Reply:
Thanks I couldn't figure how to put it into next line it was giving me an error